### DISCUSSION TITLE
Monolog configuration

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#45328] [Q&A]
I found the following link which is about `Monolog` config:

https://github.com/symfony/monolog-bundle/blob/4054b2e940a25195ae15f0a49ab0c51718922eb4/DependencyInjection/Configuration.php

But I do not understand its parameters such as `firephp` and `fingers_crossed` and `deduplication` and ..., when I should use them. Can anyone guide me on this?

**Comment by helmis123** [comment]
U can use Firephp like this
```
    public function index(LoggerInterface $logger): Response
    {
        $logger->setHandlers([new FirePHPHandler()]);
        $logger->log(LogLevel::DEBUG, 'Cocolico');
        ...
```

**Comment by mariacleh7** [comment]
Monolog is a logging library for PHP 5.3 used by Symfony2. It is inspired by the Python LogBook library. [Visit ](https://gbpick.net/ogwhatsapp/)Python LogBook to solve your problem.
